backend/models/time.py

```diff
@@ -121,8 +106,81 @@ def closest_with_early_exit(self, origin, point_list, exit_distance):
 
         return self.closest(origin=origin, point_list=relevant_points)
 
+    def closest(self, origin, point_list):
+        """
+        Find closest point in a list of points and returns min_measurement, min_point.
+
+        min_measurement is returned in minutes.
+        """
+        distance_responses = self.measure_one_to_many_points(origin=origin, point_list=point_list)
 
-class MapBoxDrivingTime(Measurer):
+        min_idx, min_measurement = min(
+            enumerate(distance_responses),
+            key=operator.itemgetter(1)
+        )
+        return float(min_measurement) / 60.0, point_list[min_idx]
+
+
+class OSRMDrivingTime(ApiDrivingTime):
+    """
+    Class for OSRM driving time measurements.
+
+    Uses the `table` OSRM API endpoint.
+    """
+
+    def __init__(self, api_url=None, early_exit_outer_radius=120.0 * 10**3):
+        """Initialize the distance class with the OSRM URL."""
+        # FIXME: Once the API accepts distances, enable a `distance` mode.
+        logger.debug('Initializing OSRM measurer.')
+        super().__init__()
+        self.api_url = api_url or os.getenv('OSRM_URL')
```

Comment:
Do the other measurers not need an `api_url`? Consider moving this to the parent class.